lib/chai/config.js

```diff
@@ -64,11 +64,31 @@ module.exports = {
    *     chai.config.useProxy = false;  // disable use of Proxy
    *
    * This feature is automatically disabled regardless of this config value
-   * in environments that don`t support proxies.
+   * in environments that don't support proxies.
    *
    * @param {Boolean}
    * @api public
    */
 
-  useProxy: true
+  useProxy: true,
+
+  /**
+   * ### config.proxyExcludedKeys
+   *
+   * User configurable property, defines which properties should be ignored
+   * instead of throwing an error if they do not exist on the assertion.
+   * This is only applied if the environment Chai is running in supports proxies and
+   * if the `useProxy` configuration setting is enabled.
+   * By default, `then` and `inspect` will not throw an error if they do not exist on the
+   * assertion object because we should not throw errors on Symbol properties such as
+   * `Symbol.toStringTag` nor on `.then` because it is necessary for for promise type-checking.
```

Comment:
The part about Symbol properties and `Symbol.toStringTag` can remain in proxify.js because that's where the type check is; it's different than the configurable excluded keys. A new note needs to be added here about `.inspect`; it's ignored by default because that property is read by `util.inspect`, for example when using `console.log` on the assertion object.